### PR TITLE
Add some examples to make it more concrete

### PR DESCRIPTION
**Pull request by standage**
The first time I presented the rectangle overlay TDD exercise, the audience was just starting to wrap their mind around the concept. The second time, I included some concrete examples in Python code. I think this helped a lot with comprehension.

### CODE REVIEW

File: short/index.html [hunk 1]
```diff
@@ -298,6 +298,35 @@
 </section>
 
 <section data-markdown>
+**Exercise**
+
+```python
+def test_overlay_no_overlap():
+    rect1 = Rectangle(0, 0, 1, 1)
```
Comment (gvwilson) [inline short/index.html:305]: I know the right way to do this is with classes, but I always use flat lists [x0, y0, x1, y1] to reduce cognitive load.

Comment (gvwilson) [reply short/index.html:305]: I think these tests will be a lot more comprehensible to people who aren't fluent in Python if they're accompanied by Cartesian grid diagrams showing the test cases.

File: short/index.html [hunk 2]
```diff
@@ -298,6 +298,35 @@
...
+    rect2 = Rectangle(2, 0, 2, 2)
+    assert overlay(rect1, rect2) == None
+
+def test_overlay_contains():
+    rect1 = Rectangle(0, 0, 5, 5)
+    rect2 = Rectangle(2, 2, 3, 3)
+    rect_ov = overlay(rect1, rect2)
+
+    assert rect_ov.x0 == 0
```
Comment (gvwilson) [inline short/index.html]: assert rect_ov == Rectangle(0, 0, 5, 5) (or if you're using lists, test for [0, 0, 5, 5])

File: short/index.html [hunk 3]
```diff
@@ -298,6 +298,35 @@
...
+    assert rect_ov.x1 == 5
+    assert rect_ov.y1 == 5
+
+def test_overlay_area():
+    rect1 = Rectangle(0, 0, 5, 5)
+    rect2 = Rectangle(0, 1, 5, 2)
+    rect_ov = overlay(rect1, rect2)
+
+    assert rect_ov.area() <= rect1.area()
+    assert rect_ov.area() <= rect2.area()
```
Comment (gvwilson) [inline short/index.html]: Why is this an informative test?

Comment (gvwilson) [reply short/index.html]: I think it's really important to have tests for rectangles that overlap on an edge and at a point - corner cases (literally).

Comment (standage) [reply short/index.html]: > Why is this an informative test?

One of the Davis attendees suggested the test. The overlap of two rectangles cannot have an area greater than either of the original triangles.

Comment (standage) [reply short/index.html]: > I think it's really important to have tests for rectangles that overlap on an edge and at a point - corner cases (literally).

This was brought up by attendees in both meetings, and I was prepared to mention these cases if nobody else did. This slide was just showing some examples, but I agree the edge cases are more interesting/important than the area test.

File: short/index.html (hunk 2 above)
Comment (standage) [reply short/index.html]: Done.

File: short/index.html (hunk 1 above)
Comment (standage) [reply short/index.html:305]: Agreed, but I'll defer on creating the figures.
